erpnext/accounts/doctype/payment_reconciliation/payment_reconciliation.json

```diff
@@ -39,6 +39,7 @@
   {
    "fieldname": "company",
    "fieldtype": "Link",
+   "ignore_user_permissions": 1,
```

Comment:
If you do this you check perm on company when the tool runs. 

Alternatively, let's disable strict user permission on single doctypes? 


@surajshetty3416 any opinion on this?